registry.yaml

```diff
@@ -81,7 +81,10 @@
   versions:
     - version: 0.1.0
       image: cmbucicd/wavefront-actions:0.1.0
+      changeLog: "adding features to send metrics to wavefront"
+      description: "adding features to send metrics to wavefront"
```

Comment:
why do we need additional description here as changeLog already covers the same?